hackathon_site/dashboard/frontend/src/pages/IncidentForm/IncidentForm.tsx

```diff
@@ -0,0 +1,255 @@
+                                    <form
+                                        onSubmit={handleSubmit}
+                                        className={styles.form}
+                                    >
+                                        <div className={styles.formContainer}>
+                                            <FormControl error={!!errors?.state}>
+                                                <Field name="state">
+                                                    {({ field }: { field: any }) => (
+                                                        <>
+                                                            <RadioGroup
+                                                                {...field}
+                                                                value={field.value}
+                                                                onChange={
+                                                                    field.onChange
+                                                                }
+                                                                data-testid="radio-state"
+                                                                row
+                                                            >
+                                                                <FormControlLabel
+                                                                    value="broken"
+                                                                    control={
+                                                                        <Radio color="primary" />
+                                                                    }
+                                                                    label="Broken"
+                                                                />
+                                                                <FormControlLabel
+                                                                    value="lost"
+                                                                    control={
+                                                                        <Radio color="primary" />
+                                                                    }
+                                                                    label="Lost"
+                                                                />
+                                                                <FormControlLabel
+                                                                    value="other"
+                                                                    control={
+                                                                        <Radio color="primary" />
+                                                                    }
+                                                                    label="Other"
+                                                                />
```

Comment:
This can be refactored using the `map() function which can be used to iterate over an array and generate the corresponding `FormControlLabel `components.

Helpful links:
https://legacy.reactjs.org/docs/lists-and-keys.html 
https://www.pluralsight.com/guides/how-to-use-the-map()-function-to-export-javascript-in-react